doc/fluidsettings.xml

```diff
@@ -418,7 +418,7 @@ and commit the results. Refresh with the following command:
             <min>64</min>
             <max>8192</max>
             <desc>
-                The size of the audio buffers (in frames).
+                This is the number of audio samples most audio drivers will request from the synth at one time. In other words, it's the amount of samples the synth is allowed to render in one go when no state changes (events) are about to happen. Because of that, specifying too big numbers here may cause MIDI events to be poorly quantized (=untimed) when a MIDI driver or the synth's API directly is used, as fluidsynth cannot determine when those events are to arrive. This issue does not matter, when using the MIDI player or the MIDI sequencer, because in this case, fluidsynth does know when events will be received.
```

Comment:
You may add as well that the issue doesn't matter because Fluidsynth is not anymore a real time MIDI synthesizer. Instead, Fluidsynth is now optimized for its internal MIDI file player and sequencer, and all other use cases are irrelevant.

Reply:
"Now"? This is the way fluidsynth works since at least version 1.1.3. It took 12 years until somebody complained about it. Therefore, I don't believe that using a real-time synth with an insanely high latency is an issue out there and I'm esp. not willing to change this behavior by potentially breaking other people's use-case as you suggested earlier. Sorry.

Reply:
>  potentially breaking other people's use-case as you suggested earlier. Sorry.

I've not suggested that.